kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/Food2Door/Orders/Order.java

```diff
@@ -0,0 +1,45 @@
+package com.kodilla.good.patterns.challenges.Food2Door.Orders;
+
+import com.kodilla.good.patterns.challenges.Food2Door.Providers.FoodProvider;
+
+public class Order {
+
+    private FoodProvider FoodProvider;
+    private String foodType;
+    private int quantity;
+    private double price;
+    private double orderValue = quantity * price;
```

Comment:
to jest niepoprawne. Kiedy tworzysz obiekt Oder (new Order), to quantity i price sa inicjalizowane z wartosciami 0, oraz 0.0 (ze wzgledu na domyslne wartosci dla typow prostych). Nastepnie orderValue jest wynikiem tego mnozenia (0 * 0.0) i zawsze da wartosc 0. Pozniej w konstruktorze i tak ta wartosc nadpisujesz.
Podsumowujac, jest to martwy kod